presto-spark-base/src/main/java/com/facebook/presto/spark/PrestoSparkQueryExecutionFactory.java

```diff
@@ -775,6 +795,64 @@ private void queryCompletedEvent(Optional<ExecutionFailureInfo> failureInfo)
             queryInfoOutputPath.ifPresent(path -> writeQueryInfo(path, queryInfo, queryInfoJsonCodec));
         }
 
+        private void processShuffleStats()
```

Comment:
so essentially we will also aggregate over all the shuffle stats on driver and print it out :). Do we need to display `ShuffleStats` on Spark Web UI as well?

Reply:
I thought it might be convenient if we want to figure out what were the shuffle speed of some particular task.